config/urls.py

```diff
@@ -9,6 +9,7 @@
 from rest_framework.routers import DefaultRouter
 from rest_framework.schemas import get_schema_view
 from rest_framework_swagger.renderers import OpenAPIRenderer, SwaggerUIRenderer
+from django_js_reverse import views
```

Comment:
Importing views from a minor third party package is probably not a good idea. I meant import the thing you need only, so "from django_js_reverse.views import urls_js". If you look at other views files, direct import pattern is the regular way.

Reply:
Ah, right.

Reply:
Done